? include/wx/fontmap.h.patch
? src/wxWindows.sln
? src/wxWindows.suo
? src/wxWindows.vcproj
? src/jpeg/jpeg.vcproj
? src/msw/minlib.patch
? src/png/png.vcproj
? src/regex/regex.vcproj
? src/tiff/tiff.vcproj
? src/zlib/zlib.vcproj
Index: include/wx/fontmap.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/include/wx/fontmap.h,v
retrieving revision 1.12
diff -b -u -2 -r1.12 fontmap.h
--- include/wx/fontmap.h	2003/03/17 10:33:30	1.12
+++ include/wx/fontmap.h	2003/04/27 13:05:04
@@ -21,5 +21,11 @@
 // ----------------------------------------------------------------------------
 
-#if wxUSE_FONTMAP
+#if !wxUSE_FONTMAP
+
+#if wxUSE_GUI
+#include "wx/fontenc.h"			// wxEncodingToCodepage (utils.cpp) needs wxGetNativeFontEncoding
+#endif
+
+#else
 
 #include "wx/fontenc.h"         // for wxFontEncoding
Index: src/msw/bitmap.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/src/msw/bitmap.cpp,v
retrieving revision 1.96
diff -b -u -2 -r1.96 bitmap.cpp
--- src/msw/bitmap.cpp	2003/04/14 01:03:41	1.96
+++ src/msw/bitmap.cpp	2003/04/27 13:05:09
@@ -1091,4 +1091,5 @@
 // ----------------------------------------------------------------------------
 
+#if wxUSE_PALETTE
 wxPalette* wxBitmap::GetPalette() const
 {
@@ -1096,4 +1097,5 @@
                            : (wxPalette *) NULL;
 }
+#endif
 
 wxMask *wxBitmap::GetMask() const
Index: src/msw/caret.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/src/msw/caret.cpp,v
retrieving revision 1.15
diff -b -u -2 -r1.15 caret.cpp
--- src/msw/caret.cpp	2003/01/02 00:41:44	1.15
+++ src/msw/caret.cpp	2003/04/27 13:05:09
@@ -36,4 +36,6 @@
 #include "wx/caret.h"
 
+#if wxUSE_CARET
+
 #include "wx/msw/private.h"
 
@@ -193,2 +195,4 @@
     }
 }
+
+#endif
Index: src/msw/colordlg.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/src/msw/colordlg.cpp,v
retrieving revision 1.14
diff -b -u -2 -r1.14 colordlg.cpp
--- src/msw/colordlg.cpp	2003/03/17 11:21:52	1.14
+++ src/msw/colordlg.cpp	2003/04/27 13:05:10
@@ -43,4 +43,6 @@
 #endif
 
+#if wxUSE_COLOURDLG
+
 #include <windows.h>
 
@@ -227,2 +229,3 @@
 }
 
+#endif

 	  	 
